.github/workflows/deploy.yml

```diff
@@ -0,0 +1,92 @@
+name: Trigger Target Workflow
+
+on:
+  workflow_dispatch
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - uses: actions/setup-python@v5
+        with:
+          python-version: '3.10'
+      
+      - name: Upgrade pip
+        run: |
+          python -m pip install --upgrade pip
+      
+      - name: install reproschema
+        run: |
+          pip install -U reproschema==0.7.2
+
+      - name: Run tests
+        run: |
+          reproschema --version
+
+      - name: Set up Git configuration
+        run: |
+          git rev-parse --abbrev-ref HEAD
+          git config --global user.email "github-actions[bot]@users.noreply.github.com"
+          git config --global user.name "github-actions[bot]"
+          git config --global credential.helper cache
+          git clone --depth 1 https://${{ secrets.AUTH }}@github.com/eipm/bridge2ai-redcap
+
+      - name: Get redcap csv
+        run: |
+          cd bridge2ai-redcap
+          cd data
+          cp bridge2ai_voice_project_data_dictionary.csv ../../.
+          cd ../..
+      
+      - name: Set up config 
+        run: |
+          cd bridge2ai-redcap
+          touch protocol.yaml
+          git fetch --tags
+          new_tag=$(git tag --sort=-creatordate | head -n 1)
+          echo -e protocol_name: "b2ai-redcap2rs" >> protocol.yaml
+          echo -e protocol_display_name: "Assessment Protocol" >> protocol.yaml
+          echo -e protocol_description: "This protocol is for Bridge2AI" >> protocol.yaml
+          echo -e redcap_version: "${new_tag:1}" >> protocol.yaml
+          echo "VERSION_TAG=${new_tag:1}" >> $GITHUB_ENV
+          cat protocol.yaml
+          cp protocol.yaml ..
+          cd ..
+
+      - name: Generate reporschema
+        run: |
+          rm -r b2ai-redcap2rs
+          mkdir output
+          cp bridge2ai_voice_project_data_dictionary.csv output
+          cp protocol.yaml output
+          cd output
+          reproschema redcap2reproschema bridge2ai_voice_project_data_dictionary.csv protocol.yaml
+          cp -r b2ai-redcap2rs ..
+          cd ..
+
+      - name: Commit reproschema
```

Comment:
last time satra mentioned that if the `version` is the only thing changed, we don't commit those files, right. we probably should implement the logic here

Reply:
If this is the case, is schemaVersion necessary? We set it in the yaml file. If we wish not to have it show up in the github diff, we could not update the version in the yaml and generate the reproschema with a placeholder version. The actions already keeps track of the schematic versioning and creates a tag upon completion. If we don't update the schema version, we can always see the important changes to the file.

Reply:
that's a good point, but without version can we still identify the original redcap file (I guess that's what our tag is doing)